Tests/SonarScanner.MSBuild.Tasks.IntegrationTest/SonarScanner.MSBuild.Tasks.IntegrationTest.csproj

```diff
@@ -10,6 +10,7 @@
     <PackageReference Include="MSBuild.StructuredLogger" Version="2.1.545" />
     <PackageReference Include="MSTest.TestAdapter" Version="2.2.5" />
     <PackageReference Include="MSTest.TestFramework" Version="2.2.5" />
+    <PackageReference Include="System.Text.Json" Version="4.7.2" />
```

Comment:
There are newer versions of this package, why didn't you target the latest one?

Reply:
It seemed there were breaking changes on the first try but I was wrong. I just updated to the latest.